bagit.py

```diff
@@ -784,6 +784,15 @@ def _make_tagmanifest_file(alg, bag_dir):
         for digest, filename in checksums:
             tagmanifest.write('%s %s\n' % (digest, filename))
 
+def _find_tag_files(bag_dir):
+    for dirName, subdirList, fileList in os.walk(bag_dir):
+        if not re.match(r'.*data$', dirName):
+            for filename in fileList:
+                if re.match(r'^tagmanifest-.+\.txt$', filename):
+                    continue
+                #remove everything up to the bag_dir directory
+                p=join(dirName, filename)
+                yield p[len(bag_dir)+1:]
```

Comment:
Stylistically, this should have spaces around operators (`len(bag_dir) + 1`) following PEP-8.

Logically, if I'm understanding it correctly this could also be written as `yield os.path.relpath(p, start=bag_dir)`